src/packages/pvm/jam-codec/decode-natural-number.test.ts

```diff
@@ -1,10 +1,10 @@
 import assert from "node:assert";
```

Comment:
This is a subset of PVM. I only included necessary files. The test files are included just in case we want to remove it from PVM at some point and it's worth keeping testing.